homeassistant/components/goalzero/const.py

```diff
@@ -25,6 +48,26 @@
     "inputDetected": ["Input Detected", DEVICE_CLASS_POWER, None],
 }
 
+SENSOR_DICT = {
+    "wattsIn": ["Watts In", DEVICE_CLASS_POWER, POWER_WATT, True],
+    "ampsIn": ["Amps In", DEVICE_CLASS_CURRENT, ELECTRICAL_CURRENT_AMPERE, False],
```

Comment:
As these are power/energy sensors, should we add state class as well? And maybe provide a `last_reset` attribute if possible?